src/lib/containers/entity_finder/details/view/DetailsViewTableRenderers.tsx

```diff
@@ -1,31 +1,76 @@
 import { Skeleton } from '@material-ui/lab'
-import { ColumnShape, SortOrder } from '@sage-bionetworks/react-base-table'
+import BaseTable, {
+  CallOrReturn,
+  ColumnShape,
+  SortOrder,
+} from '@sage-bionetworks/react-base-table'
 import moment from 'moment'
 import React, { SyntheticEvent, useEffect } from 'react'
 import { Form } from 'react-bootstrap'
 import SortIcon from '../../../../assets/icons/Sort'
 import { formatDate } from '../../../../utils/functions/DateFormatter'
-import { useGetVersionsInfinite } from '../../../../utils/hooks/SynapseAPI/useEntity'
+import { rebuildTooltip } from '../../../../utils/functions/TooltipUtils'
+import {
+  useGetEntity,
+  useGetVersionsInfinite,
+} from '../../../../utils/hooks/SynapseAPI/useEntity'
 import useGetEntityBundle from '../../../../utils/hooks/SynapseAPI/useEntityBundle'
 import { SMALL_USER_CARD } from '../../../../utils/SynapseConstants'
-import { EntityType, Reference } from '../../../../utils/synapseTypes'
-import { EntityBadgeIcons } from '../../../EntityBadgeIcons'
+import {
+  DatasetItem,
+  EntityType,
+  FileHandleAssociateType,
+  Reference,
+} from '../../../../utils/synapseTypes'
+import DirectDownload from '../../../DirectDownload'
+import {
+  EntityBadgeIcons,
+  ENTITY_BADGE_ICONS_TOOLTIP_ID,
+} from '../../../EntityBadgeIcons'
 import { EntityTypeIcon } from '../../../EntityIcon'
+import { EntityLink } from '../../../EntityLink'
+import IconSvg from '../../../IconSvg'
 import { SynapseSpinner } from '../../../LoadingScreen'
+import { DatasetItemsEditorTableData } from '../../../table/datasets/DatasetItemsEditor'
 import UserCard from '../../../UserCard'
 import { Checkbox } from '../../../widgets/Checkbox'
 import { NO_VERSION_NUMBER } from '../../EntityFinder'
 import { DetailsViewRowData } from './DetailsView'
 
+// TODO: Consider sharing logic with SynapseTableCell.tsx
```

Comment:
I was going to do this, but after talking to Duminda, it's not very important that we render table query data right now. Maybe later.